### PR TITLE
Documenting how to use pulplift

### LINKED ISSUE #51: Story: Install Galaxy on VMs using PyPi packages
Extend the Pulp 3 installer to include galaxy_ng. For this first pass our target platform will be VMs, and Pulp, galaxy_ng, and any dependencies will be installed from PyPi packages.

### PR DESCRIPTION
**Pull request by fabricio-aguiar**
closes #51

### Note:
Depends on having https://github.com/ansible/galaxy_ng/pull/56 merged

### CODE REVIEW

**Comment by fabricio-aguiar** [comment]
@bmbouter @daviddavis @mikedep333

File: README.md [hunk 1]
```diff
@@ -110,3 +110,39 @@ through Django shell (`./compose run --rm api manage shell`), use the following
...
+    source_dir: "/home/vagrant/devel/pulp_ansible"
+  galaxy-pulp:
+    source_dir: "/home/vagrant/devel/galaxy_ng/bindings/galaxy-pulp"
+  galaxy-ng:
+    source_dir: "/home/vagrant/devel/galaxy_ng"
+```
+
+#### 3. Choose a box
+
+You can find the available boxes on [pulplift](https://github.com/pulp/pulplift#available-boxes)
```
Comment (mikedep333) [inline README.md]: We should really document on pulplift's README.md what the 3 different types of boxes (bare distro vs sandbox vs source) are.

File: README.md [hunk 2]
```diff
@@ -110,3 +110,39 @@ through Django shell (`./compose run --rm api manage shell`), use the following
...
+## Pulp 3 Installer
+
+#### 1. Clone the GitHub repos
+
+```console
+$ git clone https://github.com/ansible/galaxy_ng.git
+$ git clone https://github.com/pulp/pulplift.git
+```
+
+#### 2. Setup pulplift
```
Comment (mikedep333) [inline README.md]: The requirements need to be installed. We cannot assume users have them.

https://github.com/pulp/pulplift#requirements

File: README.md [hunk 3]
```diff
@@ -110,3 +110,39 @@ through Django shell (`./compose run --rm api manage shell`), use the following
...
+
+```console
+$ cd pulplift
+$ git submodule update --init
+$ cp example.dev-config.yml local.dev-config.yml
```
Comment (mikedep333) [inline README.md]: The requirements need to be installed. We cannot assume users have them.

https://github.com/pulp/pulplift#requirements

File: README.md (hunk 1 above)
Comment (mikedep333) [inline README.md]: We should really document on pulplift's README.md what the 3 different types of boxes (bare distro vs sandbox vs source) are.

File: README.md [hunk 4]
```diff
@@ -110,3 +110,39 @@ through Django shell (`./compose run --rm api manage shell`), use the following
...
+```
+
+Uncomment `pulp-ansible`, `galaxy-pulp` and `galaxy-ng` on local.dev-config.yml:
```
Comment (mikedep333) [inline README.md]: `local.dev-config.yml` is for the `source` boxes specifically.

Comment (fabricio-aguiar) [reply README.md]: it needs to be source, because `galaxy-pulp` bindings will only exist locally

Comment (mikedep333) [reply README.md]: Also, vagrant-sshfs becomes necessary when using source boxes.

See my updated pulplift instructions. https://github.com/pulp/pulplift/pull/77/files

**Pull request merged**